.factory/automation.yml

```diff
@@ -87,23 +87,14 @@ build:
         export ARTIFACT_PASSWORD=$REPO_VATICLE_PASSWORD
         bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
         bazel run @vaticle_dependencies//distribution/artifact:create-netrc
+        bazel test //rust:typedb_driver_unit_tests --test_output=streamed || exit 1
+        export CORE_FAILED=
         tool/test/start-core-server.sh &&
           bazel test //rust/tests --test_output=streamed --test_arg=-- \
-            --test_arg=integration::queries::core &&
-          export TEST_SUCCESS=0 || export TEST_SUCCESS=1
+            --test_arg=integration::queries::core || export CORE_FAILED=1
         tool/test/stop-core-server.sh
-        exit $TEST_SUCCESS
+        if [[ -n "$CORE_FAILED" ]]; then exit 1; fi
```

Comment:
Can we use the same style of `CORE_FAILED` for the enterprise tests below please? Then we're establishing a pattern of `export X_FAILED=... || export X_FAILED=1` and switching or exiting on the variable?
Also we should call `TEST_SUCCESS` below `ENTERPRISE_` instead of `TEST_`

Reply:
It reads a little awkward compared to `TEST_SUCCESS`, so not propagating to other jobs.